### PR TITLE
Get and set apartment installments for a customer - ASU-1278

### PR DESCRIPTION
**Pull request by juhakujala**

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -102,6 +103,7 @@
   "components.customers.CustomerTableRow.phone": "Phone",
   "components.customers.CustomerTableRow.project": "Last applied to",
   "components.customers.CustomerTableRow.nin": "National identification number",
+  "components.installments.Installments.noReservations": "No saved intallments or reservations entitling to installments",
```
Comment (tuomas777) [inline src/i18n/en.json:106]: HA!
```suggestion
  "components.installments.Installments.noReservations": "No saved installments or reservations entitling to installments",
```

**Review by tuomas777 — APPROVED** [review]
Screenshot 2022-03-10 at 17 17 17

If you first save an installment with some cents, and then come back to modify it, the arrow keys work a bit wonky, and it is then possible to get into the situation shown in the above screenshot 🙂. However, I don't think too much effort should be put into that now, because it is a bit unclear ATM whether we even need to deal with cent values in that form.

In general looks good and seems to work well (enough 😉), great job! 👍 💯 🥇

File: src/i18n/en.json (hunk 1 above)
Comment (juhakujala) [reply src/i18n/en.json:106]: There's always that one typo 🤦‍♂️

**Comment by juhakujala** [comment]
> If you first save an installment with some cents, and then come back to modify it, the arrow keys work a bit wonky.

Yeah, the arrow keys on number inputs are a bit hard. Not every browser even have those so I think it should be ok as it is.

**Pull request merged**